tests/unit_tests/tools/openapi/test_specs/zapier/apispec.json

```diff
@@ -3,7 +3,7 @@
##### The Status Key <a name=\"status-key\"></a>\n\nAll actions will contain a `status`. The status can be one of four values:\n\n`success`\n\nThe action executed successfully and found results.\n\n`error`\n\nThe action failed to execute. An `error` key will have its value populated.\n\nExample:\n\n```\n    {\n        ...\n        \"action_used\": \"Gmail: Send Email\",\n        \"result\": null,\n        \"status\": \"error\",\n        \"error\": \"Error from app: Required field \"subject\" (subject) is missing. Required field \"Body\" (body) is missing.\"\n    }\n```\n\n`empty`\n\nThe action executed successfully, but no results were found. This status exists to be explicit that having an empty `result` is correct.\n\n`preview`\n\nThe action is a preview and not a real execution. A `review_url` key will contain a URL to optionally execute the action from a browser,\nor just rerun without the `preview_only` input parameter.\n\nExample:\n\n```\n    {\n        ...\n        \"action_used\": \"Slack: Send Channel Message\",\n        \"input_params\": {\n            \"Channel\": \"fun-zapier\",\n            \"Message_Text\": \"Hello World\"\n        },\n        \"review_url\": \"https://nla.zapier.com/execution/01GW2E2ZNE5W07D32E41HFT5GJ/?needs_confirmation=true\",\n        \"status\": \"preview\",\n    }\n```\n\n##### The Result Key <a name=\"result-key\"></a>\n\nAll actions will return trimmed `result` data. `result` is ideal for humans and language models alike! By default, `full_results` is not included but can be useful for machines (contact us if you'd like access to full results). The trimmed version is created using some AI and heuristics:\n\n* selects for data that is plain text and human readable\n* discards machine data like IDs, headers, etc.\n* prioritizes data that is very popular on Zapier\n* reduces final result into about ~500 words\n\nTrimmed results are ideal for inserting directly back into the prompt context of a large language models without blowing up context token window limits.\n\nExample of a trimmed results payload from \"Gmail: Find Email\":\n\n```\n    {\n        \"result\": {\n            \"from__email\": \"mike@zapier.com\",\n            \"from__name\": \"Mike Knoop\",\n            \"subject\": \"Re: Getting setup\",\n            \"body_plain\": \"Hi Karla, thanks for following up. I can confirm I got access to everything! ... Thanks! Mike\",\n            \"cc__emails\": \"bryan@zapier.com, wade@zapier.com\"\n            \"to__email\": \"Mike Knoop\",\n        }\n    }\n```\n## Changelog <a name=\"changelog\"></a>\n\n**Mar 20, 2023**\nShipped two minor but breaking changes, and one other minor change to the API's response data:\n\n* Route: `/api/v1/configuration-link/`\n  * Key `url` is now `configuration_link` **(breaking change)**\n* Route: `/api/v1/exposed/{exposed_app_action_id}/execute/`\n  * Key `rating_url` is now `review_url` **(breaking change)**\n* Route: `/api/v1/exposed/`\n  * Added `configuration_link` key"
```

Comment:
oh -- should may be this file be skipped?

Reply:
good point! will skip test data files